android/assets/jsons/Civ V - Gods & Kings/Nations.json

```diff
@@ -650,8 +650,8 @@
 		"startBias": ["Jungle"],
 		"preferredVictoryType": "Domination",
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "O divine Montezuma, your people welcomes you! May your magnificence inspire any living things! The mighty Aztecs were nomads until the 12th century, when they choose to settle in the mesa central, which will be called Mexico centuries later. Wars were frequent with other tribes to control the rich land surrounding the sacred lakes of Texcoco, Xaltocan and Zampango. In some 200 years, double-sided alliances and martial prowess made of the Aztecs the dominant power in the Central American basin, forming a mighty empire stretching from sea to sea. Alas, the empire fell under the assault of foreign devils - the Spaniards and many curses they brought with them! Wars between tribes and weaponry of doom used by the invaders made crumble the great Aztec civilization in only a few decades.",
```

Comment:
Your people welcome you!
Inspire all living things
Which would later be called Mexico
made the great Aztec civilization crumble